compass/landice/tests/ismip6_forcing/shelf_collapse/process_shelf_collapse.py

```diff
@@ -0,0 +1,281 @@
+from compass.landice.tests.ismip6_forcing.atmosphere.create_mapfile_smb \
+    import build_mapping_file
+from mpas_tools.io import write_netcdf
+from mpas_tools.logging import check_call
+from compass.step import Step
+
+
+class ProcessShelfCollapse(Step):
+    """
+    A step for processing (combine, remap and rename) the ISMIP6 shelf-collapse
+    mask data
+    """
+
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.landice.tests.ismip6_forcing.shelf_collapse.
+                    ShelfCollapse
+            The test case this step belongs to
+        """
+        super().__init__(test_case=test_case, name="process_shelf_collapse",
+                         ntasks=4, min_tasks=1)
+
+    def setup(self):
+        """
+        Set up this step of the test case
+        """
+        config = self.config
+        section = config["ismip6_ais"]
+        base_path_ismip6 = section.get("base_path_ismip6")
+        base_path_mali = section.get("base_path_mali")
+        period_endyear = section.get("period_endyear")
+        model = section.get("model")
+        scenario = section.get("scenario")
+        res_ismip6 = section.get("res_ismip6")
+        mali_mesh_name = section.get("mali_mesh_name")
+        mali_mesh_file = section.get("mali_mesh_file")
+
+        model_list_2300 = ["CCSM4", "CESM2-WACCM","HadGEM2-ES","UKESM1-0-LL"]
+        if period_endyear != 2300 and model not in model_list_2300:
+            raise ValueError(f"ice shelf-collapse masks are provided by the "
+                             f"ISMIP6 only for CCSM4-RCP85, "
+                             f"CESM2-WACCM-SSP585,HadGEM2-ES-RCP85, "
+                             f"UKESM1-0-LL-SSP585 model&scenarios. Please "
+                             f"set up the correct config options in "
+                             f"the config file.")
+
+        input_file = self._files[period_endyear][model][scenario] \
+                     [res_ismip6][0]
+        self.add_input_file(filename=mali_mesh_file,
+                            target=os.path.join(base_path_mali,
+                                                mali_mesh_file))
+        self.add_input_file(filename=os.path.basename(input_file),
+                            target=os.path.join(base_path_ismip6,
+                                                input_file))
+        self.add_output_file(filename=f"{mali_mesh_name}_"
+                                      f"{os.path.basename(input_file)}")
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        logger = self.logger
+        config = self.config
+        section = config["ismip6_ais"]
+        period_endyear = section.get("period_endyear")
+        output_base_path = section.get("output_base_path")
+        model = section.get("model")
+        scenario = section.get("scenario")
+        res_ismip6 = section.get("res_ismip6")
```

Comment:
Following up - it looks like the 4km PR has already been merged, and this option just needs to be added to the default cfg file in these test cases.

Reply:
But it also seems that the `res_ismip6` option is not being applied in the ocean_basal and ocean_thermal test cases, unless I'm missing someething.  It is in the atmosphere test case, though.  It would be good to fix that up in this PR.

Reply:
Ah yeah, I remember having to make symlinks to directories so that I could have both `Ocean_Forcing` and `Ocean_forcing`, depending on which resolution I was using.